mw-webapp/src/model/DTOModel/CommentDTO.ts

```diff
@@ -1,16 +1,16 @@
 import {z} from "zod";
 
-export const MentorCommentDTOSchema = z.object({
+export const CommentDTOSchema = z.object({
 
   /**
    * Comment's UUID
    */
   uuid: z.string(),
 
   /**
-   * Mentor's UUID @User.uuid
+   * Mentor's and way owner's UUID @User.uuid
    */
-  mentorUuid: z.string(),
+  ownerUuid: z.string(),
```

Comment:
should we call it commentatorUUID? Why it is owner if it could be mentor?

Reply:
Renamed